app/services/movie_data_service.rb

```diff
@@ -14,6 +14,7 @@ module MovieDataService
   ["unwatched movies", "unwatched movies"], ["only show unwatched", "only show unwatched"],
   ["only show watched", "only show watched"], ["movies not on a list", "movies not on a list"] ]
 
+  # TODO: move these into a TMDB::MenuOptions module
```

Comment:
I believe this is a list of Genres and their TMDB ids. I'd like this to go into some sort of `TMDB` namespaced object in order to make that more clear.